### PR TITLE
Added internationalization with react-i18next

### PR DESCRIPTION
**Pull request by guvencb**
Issue #6 

- Added react-i18next
- Hooked up existing components with react-i18next
- Added sample translations for EN/NL
- Created a dummy language changer component and added it to navbar

### CODE REVIEW

File: public/locales/nl/translation.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "login": "Log in",
+    "home": "Huis",
```
Comment (timstokman) [inline public/locales/nl/translation.json:3]: I'd just call this home. Don't think "Thuis" will work.

**Comment by timstokman** [comment]
How is the default translation setup? Does it just take the browser language setting?

**Review by timstokman — APPROVED** [review]
Approved. But look at the comments I had?

**Comment by guvencb** [comment]
@timstokman it's cookie based and creates the initial cookie from the fallback language

**Pull request merged**